plugins/org.jkiss.dbeaver.erd.ui/src/org/jkiss/dbeaver/erd/ui/notations/ERDNotation.java

```diff
@@ -52,4 +52,12 @@ public void applyNotationForEntities(
         @NotNull ERDAssociation association,
         @NotNull Color bckColor,
         @NotNull Color frgColor);
+
+    /**
+     * Indentation value is a length of orthogonal line for source and target
+     * decorator. Each notation keeps own value to handle require behavior.
+     *
+     * @return - value of indentation
+     */
+    public double getIndentation();
```

Comment:
This javadoc makes no sense, unfortunately. The terms "identation" and "length" mean different things.